### ISSUE TITLE
onEvents/onSubscribe nothing happening

### ISSUE THREAD

**Issue by rommansabbir** [parse-community/ParseLiveQuery-Android#95]
Here is my code: 

        Parse.initialize(new Parse.Configuration.Builder(this)
                .applicationId(getString(R.string.back4app_app_id))
                // if defined
                .clientKey(getString(R.string.back4app_client_key))
                .server(getString(R.string.back4app_server_url))
                .build()
        );

        ParseInstallation.getCurrentInstallation().saveInBackground();

Main Activity:

        ParseLiveQueryClient parseLiveQueryClient = null;
        Log.d(TAG, "onCreate: ");
        parseLiveQueryClient = ParseLiveQueryClient.Factory.getClient();
        ParseQuery<ParseObject> parseQuery = ParseQuery.getQuery("Accident");

        SubscriptionHandling<ParseObject> subscriptionHandling = parseLiveQueryClient.subscribe(parseQuery);
        subscriptionHandling.handleSubscribe(new SubscriptionHandling.HandleSubscribeCallback<ParseObject>() {
            @Override
            public void onSubscribe(ParseQuery<ParseObject> query) {
                Log.d(TAG, "onSubscribe: ");
            }
        });
        subscriptionHandling.handleEvents(new SubscriptionHandling.HandleEventsCallback<ParseObject>() {
            @Override
            public void onEvents(ParseQuery<ParseObject> query, SubscriptionHandling.Event event, ParseObject object) {
                Log.d(TAG, "onEvents: ");
            }
        });

not getting any response or error from subscription. live query enabled on the server settings.

**Comment by mmimeault** [comment]
Hello, did you find the issue/solution? If so, can you give some detail. Sometime it can help others that fall into the same problem. Thanks

**Comment by rommansabbir** [comment]
Yes, sure. Currently, I'm working on it. I will post the github link here ASAP.

**Comment by rommansabbir** [comment]
This is a working project. Check this out
https://github.com/rommansabbir/ParseLiveQueryAndroid/
